src/containers/App/App.js

```diff
@@ -19,41 +22,55 @@ if (!global.Intl) {
 class AppContainer extends Component {
   render() {
     return (
-      <div className="site-wrapper" style={{ paddingBottom: '20px' }}>
-        <LayoutContainer>
-          <LayoutRow>
-            <Header />
-          </LayoutRow>
-
-          {isLoggedIn()
-            ? <LayoutRow>
-                <LayoutColumn width={1 / 1}><AppNavigation /></LayoutColumn>
-              </LayoutRow>
-            : null}
+      <StyleProvider
+        cssNode={null}
+        fontNode={null}
+        dev={false}
+        selectorPrefix="cf-"
+      >
+        <div className="site-wrapper" style={{ paddingBottom: '20px' }}>
+          <LayoutContainer>
+            <LayoutRow>
+              <Header />
+              {isLoggedIn() ? null : <GradientBanner />}
+            </LayoutRow>
+            <div
+              className="content-wrapper"
+              style={isLoggedIn() ? null : { marginTop: '-105px' }}
+            >
+              {isLoggedIn()
+                ? <LayoutRow>
+                    <LayoutColumn width={1 / 1}><AppNavigation /></LayoutColumn>
+                  </LayoutRow>
+                : null}
 
-          {isLoggedIn()
-            ? <LayoutRow>
-                <LayoutColumn width={2 / 20}>&nbsp;</LayoutColumn>
-                <LayoutColumn width={16 / 20}>
-                  {this.props.children}
+              {isLoggedIn()
+                ? <LayoutRow>
+                    <LayoutColumn width={2 / 20}>&nbsp;</LayoutColumn>
+                    <LayoutColumn width={16 / 20}>
+                      {this.props.children}
+                    </LayoutColumn>
+                    <LayoutColumn width={2 / 20}>&nbsp;</LayoutColumn>
+                  </LayoutRow>
+                : <LayoutRow>{this.props.children}</LayoutRow>}
+              <LayoutRow>
+                <LayoutColumn width={1 / 1}>
+                  <p style={{ textAlign: 'center' }}>
```

Comment:
There are a few more places were `<p>` or  `<a>` is used rather than the component. @jwineman is it worth to change every time someone uses non cf-ui elements?